### ISSUE TITLE
Steamshovel tries to build without GLUT (Trac #946)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#940]
Steamshovel tries to build, then fails because GLUT is not installed.  It shouldn't try to build if GLUT's not there.

Migrated from https://code.icecube.wisc.edu/ticket/946
```json
{
    "status": "closed", 
    "changetime": "2015-04-22T21:28:00", 
    "description": "Steamshovel tries to build, then fails because GLUT is not installed.  It shouldn't try to build if GLUT's not there.", 
    "reporter": "richman", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1429738080648709", 
    "component": "cmake", 
    "summary": "Steamshovel tries to build without GLUT", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-04-22T21:05:55", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150422T21:06:48: richman changed attachment from "" to "cmake_log_noGLUT.txt"

**Comment by icecube-trac** [comment]
Trac update at 20150422T21:07:42: richman changed attachment from "" to "make_log_noGLUT.txt"

**Comment by icecube-trac** [comment]
Trac update at 20150422T21:23:20: david.schultz commented:

> Steamshovel should probably check cmake for OPENGL_GLU_FOUND

**Comment by icecube-trac** [comment]
Trac update at 20150422T21:28:00: 
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> In changeset 131913:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="131913"
> fixes #940 by actually checking for GLU.
> ```


* david.schultz changed status from "new" to "closed"
* david.schultz changed owner from "" to "david.schultz"
